mkosi.conf.d/arch/mkosi.conf

```diff
@@ -34,6 +35,8 @@ Packages=
         tpm2-tools
         tpm2-tss
         vim-minimal
+        wget
```

Comment:
i'd really drop wget. let's focus on curl for everything. "curl -O" is just as good wget